examples/MDM_V3_Library4x4/MDM_V3_Library4x4.ino

```diff
@@ -0,0 +1,100 @@
+/********************************************************************
+ *                       MiDispositivoMIDI V3                       *
+ ********************************************************************
+ * 2018 BanaNut Electronics                                         *
+ * bananutelectronics@gmail.com                                     *
+ *                                                                  *
+ * See license and documentation in the repository.                 *
+ * https://github.com/bananut-electronics                           *
+ *                                                                  *
+ * Code:     MDM_V3_Library4x4.ino                                      *
+ * Hardware: Single 4x4 boards configuration        *
+ * Brief:    This code handles a MiDispositivoMIDI V3 with three    *
+ *           expansions, which is, four devices in total. They have *
+ *           to be connected in a 4x4, creating an array of 16     *
+ *           buttons. With this code you can set a default color    *
+ *           when the button is an idle state, and another color    *
+ *           when the button is pushed. There is also a subroutine  *
+ *           that does some animation with the leds.                *
+ *                                                                  *
+ ********************************************************************/
+
+// Include the library
+#include <MiDispositivoMIDI_V3.h>
+
+// Creates a mdm with four pages and only one expansion
+BoardRoles bRoles[1] = {TOUCHPAD};
+MiDispositivoMIDI_V3 mdm = MiDispositivoMIDI_V3(SINGLE_DEVICE_4X4, bRoles);
+
+// Couple of variables for the led animation
+int buttonCounter    = 0;
+int expansionCounter = 0;
```

Comment:
Since this code is made for only 4x4 board with no expansions, we can get rid of "expansionCounter" variable